R/runClusterSummaryMetrics.R

```diff
@@ -40,16 +44,24 @@ runClusterSummaryMetrics <- function(inSCE, useAssay="logcounts", featureNames,
     warning("Specified genes '", toString(falseGenes), "' not found in ", warning)
   }
   
+  tempSCE <- inSCE[featureNames, ]
+
+  
+  if(isTRUE(scale)){
+    runNormalization(inSCE=tempSCE, useAssay=useAssay,scale = TRUE, normalizationMethod = NULL, transformation = NULL,
+                     pseudocountsBeforeNorm = NULL, pseudocountsBeforeTransform = NULL)
+    useAssay <- "counts"
```

Comment:
The runNormalization function will essentially return a SCE object with a new assay specified by the `outAssayName` parameter. So, you should use that downstream. 

You can see more information about the function here: https://rdrr.io/github/compbiomed/singleCellTK/man/runNormalization.html